share/da.po

```diff
@@ -4,139 +4,139 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Project-Id-Version: 0.0.5\n"
-"PO-Revision-Date: 2017-12-06\n"
-"Last-Translator: mats.dufberg@iis.se\n"
+"PO-Revision-Date: 2019-07-08\n"
+"Last-Translator: haarbo@dk-hostmaster.dk\n"
 "Language-Team: Zonemaster Team\n"
 "MIME-Version: 1.0\n"
 
 msgid "Print version information and exit."
-msgstr "Print version information and exit."
+msgstr "Udskriv version og afbryd."
 
 msgid "The minimum severity level to display"
-msgstr "The minimum severity level to display"
+msgstr "Det mindste alvorlighedsniveau som skal udskrives."
 
 msgid "The locale to use for messages translation."
-msgstr "The locale to use for messages translation."
+msgstr "Landestandarden som skal bruges ved udskrift."
```

Comment:
The term "locale" is special and probably not translatable (I notice that the Swedish translation is strange too). I suggest something like (please correct my Danish):
```
"Den 'locale' som skal bruges ved oversættelse af beskeder."
```